powerline_shell/segments/ruby_version.py

```diff
@@ -10,10 +10,10 @@ def add_to_powerline(self):
             p1 = subprocess.Popen(["ruby", "-v"], stdout=subprocess.PIPE)
             p2 = subprocess.Popen(["sed", "s/ (.*//"], stdin=p1.stdout, stdout=subprocess.PIPE)
             version = p2.communicate()[0].decode("utf-8").rstrip()
-            if os.environ.has_key("GEM_HOME"):
-              gem = os.environ["GEM_HOME"].split("@")
-              if len(gem) > 1:
-                version += " " + gem[1]
+            if "GEM_HOME" in os.environ.keys():
```

Comment:
Can this not be `"GEM_HOME" in os.environ`?

Reply:
Actually, I find this version of the code even nicer:

```python
gem = os.getenv("GEM_HOME", "").split("@")
if len(gem) > 1:
    ...
```

Reply:
Your very right. I was just patching the issue for the python error. I'll fix it up and make it nice and not just functional.